ops/ccs-ops-misc/securityhub/resolve-inactive-findings/resolve-inactive-findings.py

```diff
@@ -0,0 +1,530 @@
+            topics.append(get_id_from_arn(topic['TopicArn']))
+    return topics
+
+
+# Get active IAM users
+def get_active_iam_users(client):
+    users = client.list_users()
+    return [user['UserName'] for user in users['Users']]
+
+
+# Get active ec2 security groups
+def get_active_security_groups(client):
+    groups = []
+    paginator = client.get_paginator('describe_security_groups')
+    for page in paginator.paginate():
+        for group in page['SecurityGroups']:
+            groups.append(group['GroupId'])
+    return groups
+
+
+# Get active lambda functions
+def get_active_lambdas(client):
+    lambdas = []
+    paginator = client.get_paginator('list_functions')
+    for page in paginator.paginate(FunctionVersion='ALL',):
+        for lambda_function in page['Functions']:
+            lambdas.append(lambda_function['FunctionName'])
+    return lambdas
+
+
+# Get active IAM access keys
+def get_active_iam_access_keys(client):
+    keys = []
+    user_paginator = client.get_paginator('list_users')
+    for user_page in user_paginator.paginate():
+        for user in user_page['Users']:
+            key_paginator = client.get_paginator('list_access_keys')
+            for key_page in key_paginator.paginate(UserName=user['UserName']):
+                for key in key_page['AccessKeyMetadata']:
+                    keys.append(f"AWS::IAM::AccessKey:{key['AccessKeyId']}")        
+    return keys
+
+
+# Get active rds instances
+def get_active_rds_instances(client):
+    instances = []
+    paginator = client.get_paginator('describe_db_instances')
+    for page in paginator.paginate():
+        for instance in page['DBInstances']:
+            instances.append(instance['DBInstanceIdentifier'])
+    return instances
+    
+
+# Get active ASG groups
+def get_active_asg_groups(client):
+    groups = []
+    response = client.describe_auto_scaling_groups()
+    for asg in response['AutoScalingGroups']:
+        groups.append(asg['AutoScalingGroupName'])
+    return groups
+
+
+# Get active IAM policies
+def get_active_iam_policies(client):
+    policies = []
+    paginator = client.get_paginator('list_policies')
+    for respose in paginator.paginate(Scope='Local', PolicyUsageFilter='PermissionsPolicy'):
+        for policy in respose['Policies']:
+            policies.append(policy['PolicyName'])
+    return policies
+
+
+# Get active RDS cluster snapshots
+def get_active_rds_snapshots(client):
+    snapshots = []
+    paginator = client.get_paginator('describe_db_cluster_snapshots')
+    for response in paginator.paginate():
+        for snapshot in response['DBClusterSnapshots']:
+            snapshots.append(snapshot['DBClusterSnapshotIdentifier'])
+    return snapshots
+
+
+# Returns a list of active ec2 volumes
+def get_active_volumes(client):
+    # TODO: add pagination
+    active_volumes = []
+    response = client.describe_volumes(
+        Filters=[{'Name': 'status', 'Values': list(ACTIVE_VOLUME_STATES)}]
```

Comment:
RE: **set-to-list operation (2/3)**
```suggestion
        Filters=[{'Name': 'status', 'Values': ACTIVE_VOLUME_STATES}]
```